src/main/java/com/gl/donate_receive/controller/ItemController.java

```diff
@@ -73,6 +79,11 @@ public String getManagingForm(@PathVariable("itemId") String itemId, Model model
 	public String getById(@PathVariable("itemId") String itemId, Model model) {
 		var item = itemService.getById(itemId);
```

Comment:
What happens if the item was not found?

Reply:
> What happens if the item was not found?

In this case user will be redirected to error page with info:
404 / Not Found
Item with id: 014bd2be-64d7-45f8-8d9e-7b4d6d2b4c4 not found